cmd/skr/README.md

```diff
@@ -92,3 +95,43 @@ Upon error, the `key/release` POST method response carries a `StatusForbidden` h
 }
 ```
 
+Additionally, the ```skr``` tool instantiates a GRPC server that can be accessed by running grpcurl commands. The GRPC server exposes the following methods:
+
+API
+---
+
+The `<ip:port> list` command lists the services exposed on a specific IP address and port.
+
+```bash
+grpcurl -v -plaintext 127.0.0.1:50000 list
+```
+
+The `<ip:port> list <service-name>` command lists the exposed APIs under a specific service on a specfic IP address and port.
+
+```bash
+grpcurl -v -plaintext 127.0.0.1:50000  list keyprovider.KeyProviderService
+```
+
+The SayHello method of the KeyProviderService is used to test whether APIs under KeyProviderService can be reached.

```

Comment:
for style consistency:
SayHello --> `SayHello`
GetReport --> `GetReport`
UnWrapKey --> `UnWrapKey`